clickhouse-core/src/main/scala/xenon/clickhouse/parse/AstVisitor.scala

```diff
@@ -83,7 +83,7 @@ class AstVisitor extends ClickHouseSQLBaseVisitor[AnyRef] with Logging {
           _ttl = ttlOpt,
           _settings = settings
         )
-      case eg: String if "ReplicatedMergeTree" equalsIgnoreCase eg =>
+      case eg: String if ("ReplicatedMergeTree" equalsIgnoreCase eg) || ("SharedMergeTree" equalsIgnoreCase eg) =>
```

Comment:
"SharedMergeTree" looks like a new concept added recently, obviously, it needs a  `SharedMergeTreeEngineSpec`

Reply:
constructing `ReplicatedMergeTreeEngineSpec` from a "SharedMergeTree" is incorrect, and there is no test coverage for "SharedMergeTree"